### DISCUSSION TITLE
Crazy Station Idea - Citadel

### DISCUSSION THREAD

**Discussion by everydayeldergod** [space-wizards/space-station-14#33973] [Ideaguys]
I was playing on Barratry the other day, thinking about how it was kind of fun to be able to repair a station and make it more livable. On Barratry, this means making it LOOK less like an active warzone. And then I thought to myself: what if there was a station that was so mechanically difficult that it needed to be rebuilt over the course of the round.

Thus, I present my idea: Citadel Station. Citadel Station was a failed outpost of Nanotrasen, destroyed by the Syndicate in a deadly surprise attack. However, the Station Crew have been deployed on a forward-operating drop pod to get the station up and running again.

SPECIAL CONDITIONS:
-Citadel's role assignments would be heavily weighted towards increased engineering, since a large portion of the round is spent building the station.
-It should also NEVER be in rotation, because admins would need to watch the match carefully and handle antags (read blitzop nukies) that don't mesh with the difficult nature of Citadel.
-The Drop Pod would have a fully functional atmos and engineering setup with tons of extra materials - I'm thinking hundreds of steel plates, glass sheets, plastic, wire, airlock flatpacks, door flatpacks. It would also be powered by an extensive SUPERPACMAN generator array, which could last for upwards of 30 minutes if properly managed.
-The damage to the station should be engaging to repair. For example, atmos systems would be largely intact because spending an hour setting up air alarms sounds like hell on Earth. The service department would be relatively undamaged, but engineering would require extensive floor and wall repairs. 
-Citadel would have a large solar array capable of powering the entire station once properly attached, though setup may require printing your own wires and setting up some computers.
-The headlockers would spawn in the Bridge of the Drop Pod, not in the heads offices. Lorewise, its implied that the syndicate raided the station for NT valuables before leaving.
-Fortunately for Cargo, the drop pod already has Cargo Selling Panels! No ATS needed. Unfortunately for Cargo, their department was heavily damaged by the Syndicate attack.

I want to know if this would even be considered for addition before I start learning mapping for it. I understand its quite a derivative idea.

**Comment by Andreymyski** [comment]
Best!

**Comment by ArcaneOcto** [comment]
Now this. This is epic.

**Comment by ArcaneOcto** [comment]
What would Salvage be doing? Could be interesting if they had to clear large chunks of asteroids that have collidided with the derelict station, clogging the halls. That way it's not *just* engineering who can help with the repairs.
